app/actors/iiif_print/actors/iiif_print_upload_actor.rb

```diff
@@ -28,16 +30,20 @@ def hold_upload_paths(env)
         return if upload_ids.empty?
         uploads = Hyrax::UploadedFile.find(upload_ids)
         paths = uploads.map(&method(:upload_path))
-        @pdf_paths = paths.select { |path| path.end_with?('.pdf') }
+        @pdf_paths = paths.select { |path| path.end_with?('.pdf', '.PDF') }
```

Comment:
just in case there's something weird like `a_file.Pdf` consider using case insensitive regex
```rb
@pdf_paths = paths.select { |path| path.match? /\.pdf$/i }
```

Reply:
Possibly? I was thinking a TODO may be necessary to find a way to identify PDFs differently which is why I left it this way for now. Remote URLS can contain PDF files but don't always end in .pdf (i.e. google docs links).